### PR TITLE
docs: add quickstart and quick links

### PR DESCRIPTION
**Pull request by ToeFrog**
Added a quickstart section to the README file as well as quick links to jump to specific sections in the file.

### CODE REVIEW

**Review by cmjchrisjones — CHANGES_REQUESTED** [review]
Hey @ToeFrog, can you also change the target branch from `main` to `vNext` too please?

File: README.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ A Visual Studio Code extension that listens to twitch chat and allows viewers to
...
 ![](./resources/screenshot-example.gif)
 
+<a name="quickstart"></a>
+## Quickstart Guide
+
+
+Install [VS Code](https://code.visualstudio.com/Download) if you don't already have it.
+
+Install [Twitch Themer](https://marketplace.visualstudio.com/items?itemName=MichaelJolley.vscode-twitch-themer) from the Visual Studio Marketplace
+Connect Twitch Themer to Twitch from the [Command Pallete](#command-palette) (Ctrl + Shift + P)
```
[the remark this answers is not in the document]
Comment (ToeFrog) [reply README.md]: @cmjchrisjones Updated and fixed the type. Also changed the target branch.

**Comment by cmjchrisjones** [comment]
@ToeFrog seems to be a lot more changes in this PR now than there was before, guessing that your original branch was based of `main` rather than `vNext`?

**Comment by ToeFrog** [comment]
I've created a new branch off vNext and made the changes there. I will submit a new PR and close this one. Sorry for the confusion. In the future I will make sure to target the vNext branch rather than main.

**Pull request closed without merging**